sklearn/cluster/_hdbscan/hdbscan.py

```diff
@@ -131,6 +134,7 @@ def _hdbscan_brute(
         distance_matrix = pairwise_distances(
             X, metric=metric, n_jobs=n_jobs, **metric_params
         )
+    distance_matrix /= alpha
```

Comment:
What if `alpha=None` in this case?

```suggestion
    if alpha is not None:
        distance_matrix /= alpha
```

Reply:
I just left that as it is on `hdbscan` since we'll be removing/revisiting `alpha` in the follow-up PR and this portion is in `_hdbscan_brute` and a bit out of scope for this PR.